src/test/app/Path_test.cpp

```diff
@@ -171,6 +171,131 @@ rpf(jtx::Account const& src, jtx::Account const& dst, std::uint32_t num_src)
     return jv;
 }
 
+// test helper
+bool checkArraySize(Json::Value const& val, unsigned int size)
+{
+    return val.isArray() &&
+           val.size() == size;
+}
+
+using verify_paths =
+    std::vector<std::vector<std::pair<ripple::Issue, jtx::Account>>>;
+
+// compares expected paths (pairs of Issue/Account) with
+// json output from path_find
+bool
+check_paths(
```

Comment:
Does this checker provide different functionality than the style used existing tests in this file?
```
BEAST_EXPECT(same(st, stpath("dan"), stpath("bob", "carol")));
```